### PR TITLE
Added flight choice

### PR DESCRIPTION
**Pull request by cycloop**

### CODE REVIEW

**Comment by cycloop** [comment]
Plz review this, @casey-collab

File: english/find-exit/leave.md [hunk 1]
```diff
@@ -31,3 +31,5 @@ The forest turns dark. I don't see the ice-cold water and fall into it.
 [Invite some friends over to your room](../invite-friends/friends.md)
 
 [shoot a flare to wave down the chococopter](chococopter/chococopter.md)
+
+[Wave at the chococopter](../flight/flight.md)
```
Comment (casey-collab) [inline english/find-exit/leave.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
It looks like the file `english/flight/flight.md.txt` is empty.  Please make sure you don't add empty files to the repository.  You could add one sentence to this file, or delete the option that leads to it.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cycloop** [comment]
At @casey-collab

**Comment by casey-collab** [comment]
Looks great! :rabbit:

**Pull request merged**